modules/exploits/windows/misc/crosschex_device_bof.rb

```diff
@@ -0,0 +1,81 @@
+
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+  PAYLOAD_MAX_LENGTH = 8947
+  PACKET_LEN = 10
+
+  include Msf::Exploit::Remote::Udp
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Anviz CrossChex Local Buffer Overflow',
```

Comment:
What does `Local` mean here? This looks remote because of the broadcast UDP?

Reply:
Name was lifted from the [EDB article](https://www.exploit-db.com/exploits/47734), but I'll take it out as it's not making things any clearer and having googled the phrase I can't find a stated definition of a `Local Buffer Overflow`.

Reply:
If it's exploitable only from the same machine, I'd say it's local, but it seems remote.